.gitmodules

```diff
@@ -272,6 +272,12 @@
 [submodule "contrib/ulid-c"]
 	path = contrib/ulid-c
 	url = https://github.com/ClickHouse/ulid-c.git
+[submodule "contrib/limonp"]
```

Comment:
It's tempting to move new entries to the end of a list but it can also create useless merge conflicts with other PRs that do the same. To reduce that risk, I often choose a random position.